views/celebrities/new-celebrities.hbs

```diff
@@ -0,0 +1,20 @@
+<h2>New Celebrity</h2>
+
+<form action="/celebrities/create" method="POST">
+
+    <label>Name:
+        <input type="text" name="string" />
+    </label>
+
+    <label>Occupation:
+        <input type="text" name="string" />
+    </label>
+
+    <label>Catch Phrase:
+        <input type="text" name="string" />
+    </label>
+
+
+    <button type="submit">Add</button>
+
+</form>
```

Comment:
Compare this form to the form for creating movies. Example below:
```
    <label> Title:
        <input type="text" name="title" />
    </label>
```

Input attribute `name` has to match the property you are reading from req.body.
for example:
```
req.body.name = "Jim Carrey"
req.body.occupation="actor"
```
etc

At the moment you have 
```
req.body.string = "Jim Carrey"
req.body.string="actor"
```
so you are even overwriting it with every input